contrib/pkg/broker/user_provided/controller/controller.go

```diff
@@ -107,16 +109,28 @@ func (c *userProvidedController) CreateServiceInstance(
 	return &brokerapi.CreateServiceInstanceResponse{}, nil
 }
 
-func (c *userProvidedController) GetServiceInstance(id string) (string, error) {
-	return "", errors.New("Unimplemented")
+func (c *userProvidedController) GetServiceInstance(
```

Comment:
The spec does not have a GetServiceInstance, so Unimplemented should really be a 404?